packages/bsky/src/util.ts

```diff
@@ -0,0 +1,40 @@
+export type ParsedLabelers = {
+  dids: string[]
+  redact: Set<string>
+}
+
+export const parseLabelerHeader = (header: string): ParsedLabelers => {
+  const labelers = header.split(',').map((part) => part.trim())
+  const labelerDids = new Set<string>()
+  const redactDids = new Set<string>()
+  for (const labeler of labelers) {
+    if (labeler.length === 0) {
+      continue
+    }
```

Comment:
This would technically make the header invalid.  If we want to pull in a parser, `parseList()` from [structured-headers](https://www.npmjs.com/package/structured-headers) does what we need here.

Reply:
ah nice, that sounds like it's probably worth it 👍 

My thinking here was specificlaly for the case of an empty string val ie `atproto-accept-labelers: `